addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-exitstatus.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Automation Framework - exitStatus Job
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Automation Framework - exitStatus Job</H1>
+
+This job sets the exit code based on scan results.
```

Comment:
> ZAP's exit code

Reply:
Also worth linking to the main help page and link back since we mentioned there the exit values.